packages/documentation/src/pages/platform/unit-testing.mdx

```diff
@@ -0,0 +1,203 @@
+## How do unit tests work?
+
+Table of Contents
+ - [Unit](#unit)
+ - [End to End](#end-to-end)
+ - [How to debug](#how-to-debug)
+
+### Unit
+**Description**: Used to test individual functions or components.
+
+**File format**: *.unit.spec.js
+
+**Where**: Any *.unit.spec.js file located in the `/src` folder. Typically located in a directory close to the code being tested.
+
+**When**: Locally through npm script commands, Jenkins build (Unit), and After merging to master
+
+**Libraries used**: [mocha.js](https://mochajs.org/), [chai.js](https://www.chaijs.com/), [chai-as-promised](https://github.com/domenic/chai-as-promised), [sinon.js](https://sinonjs.org/), [enzyme](https://airbnb.io/enzyme/), [enzyme-adapter-react-16](https://www.npmjs.com/package/enzyme-adapter-react-16), [react-dom](https://www.npmjs.com/package/react-dom), [react-dom/test-utils](https://reactjs.org/docs/test-utils.html), [react-test-renderer](https://www.npmjs.com/package/react-test-renderer), [jsdom](https://airbnb.io/enzyme/docs/guides/jsdom.html), [mocha-junit-reporter](https://www.npmjs.com/package/mocha-junit-reporter)
+
+**Process**:
+
+1. Mocha builds target unit test files in the same environment
+2. Babel is used to transpile the unit tests
+3. CHOMA (not sure if this is a Mocha extension or something else) randomizes the test execution
+4. Unit tests are executed serially
+5. Mocha uses the [mocha-junit-reporter](https://www.npmjs.com/package/mocha-junit-reporter) reporterer to output the test results. This is used by Jenkins to for reporting.
+6. example of a success report
+7. exmaple of a failure report
+
+**Debugging**:
```

Comment:
I don't think this debugging section makes sense in here. I think it makes sense as its own common task doc or in the testing common task doc.

To me, the purpose of this doc is to describe what's happening when unit tests are run. It's about context and information, rather than showing you how to do something.

Reply:
Troubleshooting tests doc would be good- have we actually tried this process? I've never used the remote debugger on the mocha task like this.

Reply:
@rianfowler Yes it works. I've used it this way to debug multiple times. I also use VSCode also. You should try it out, it's pretty nice.